src/components/pages/Categories/List.vue

```diff
@@ -0,0 +1,54 @@
+<template>
+  <div class="Categories-edit">
+    <app-category-post class="app-category-post">
+    </app-category-post>
+    <app-category-list
+    :categories="categoriesList"
+    :theads="theads"
+    class="app-category-list"
+    >
+    </app-category-list>
+  </div>
+</template>
+
+<script>
+import { CategoryList } from '@Components/molecules'
+import { CategoryPost } from '@Components/molecules';
+import Mixins from '@Helpers/mixins';
+
+export default {
+  components: {
+    appCategoryList: CategoryList,
+    appCategoryPost: CategoryPost,
+  },
+  mixins: [Mixins],
+  data() {
+    return {
+      theads: ['カテゴリー名', '', '', '', '', ''],

```

Comment:
[imo]

```suggestion
      theads: ['カテゴリー名'],
```

1つの要素だけで良いかなと思います！

Reply:
修正しました。